src/primer_bed.cpp

```diff
@@ -85,7 +85,7 @@ void print_bed_format(){
   std::cout << "It requires the following columns delimited by a tab: chrom, chromStart, chromEnd, name, score, strand" << std::endl;
 }
 
-std::vector<primer> populate_from_file(std::string path){
+std::vector<primer> populate_from_file(std::string path, int32_t offset = 0){
```

Comment:
Given that `offset` will not be needed in a lot of other functions, can you please overload this function instead of adding offset to every call?